queue_services/account-mailer/src/account_mailer/email_templates/business_invitation_email_for_bceid.html

```diff
@@ -1,53 +1,13 @@
-<!DOCTYPE html>
-<html>
-<head>
-    <base href="/">
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="referrer" content="origin-when-cross-origin"/>
-    <meta name="author" content="BC Registries and Online Services">
-    <title> {{ title }}</title>
-    [[style.html]]
-</head>
-<body>
-<table style="width: 100%;" cellspacing="0" cellpadding="0">
-    <tr>
-        <td style="padding-left: 20px; padding-right: 20px;">
-            <table style="max-width: 640px; margin: 0 auto; border: 0; border-collapse: collapse; border-radius: 6px; background-color: #ffffff;"
-                   cellspacing="0" cellpadding="0">
-                <tr>
-                    <td style="padding-top: 30px; padding-bottom: 30px;">
-                        [[logo.html]]
-                    </td>
-                </tr>
-                <tr>
-                    <td>
-                        <h1 style="margin: 0; line-height: 34px; letter-spacing: -1px; font-family: Verdana, Arial, sans-serif; font-size: 24px; font-weight: 700;">
-                            You have been invited to join the <strong>{{ account_name }}</strong> account at Business Registry.
-                        </h1>
-                        <p style="margin-top: 25px; margin-bottom: 0; line-height: 24px; font-family: Verdana, Arial, sans-serif; font-size: 16px;">
-                            <strong>{{ user_first_name }} {{ user_last_name }}</strong> has invited you to join the <strong>{{ account_name }}</strong> account at Business Registry.
```

Comment:
This line is missing, not sure if it was intentional

Reply:
Missed it.